src/twisted/test/test_proto_helpers.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+from twisted.trial import unittest
+
+
+
+class DeprecationTests(unittest.TestCase):
+    """
+    Deprecations in L{twisted.test.proto_helpers}.
+    """
+    def test_AccumlatingProtocol(self):
+        """
+        L{proto_helpers.AccumlatingProtocol} is deprecated and
+        generates a warning.
+        """
+        from twisted.test.proto_helpers import AccumulatingProtocol
+        AccumulatingProtocol  # shush
+
+        warnings = self.flushWarnings(
```

Comment:
These tests all seem substantially duplicated.  If you're going to stick with each object being its own deprecated attribute, can you refactor these to call a single helper so we don't have so much duplication?

Reply:
Refactored these and moved them to twisted.internet.test.test_testing.DeprecationTests.